composer.json

```diff
@@ -20,6 +20,8 @@
 		],
 		"post-install-cmd": [
 			"cd php52; composer install; cd .."
-		]
+		],
+		"phpcs": "phpcs --standard=src/ruleset-wordpress.xml wpengine-phpcompat.php load-files.php src/*.php",
+		"phpcs:fix": "phpcbf --standard=src/ruleset-wordpress.xml wpengine-phpcompat.php load-files.php src/*.php"
```

Comment:
Pro-tip for both the above commands:
1. Rename the `src/ruleset-wordpress.xml` file to `phpcs.xml.dist` (in the project root).
    That way PHPCS will pick up on it automatically and you don't need to pass the `--standard`.
2. Set the files which should be scanned from within the ruleset.
    You then don't need to pass them on the command-line anymore and there can be no discrepancies ever between what `phpcs` or `phpcbf` is run against.

Reply:
I doubt that as it includes the `WordPress` coding standard which does not check compatibility....

Reply:
Yeah, I realized that last night. I am contacting the original author to see why they two files exist separately. Thank you for the diligence, @jrfnl!

Reply:
>Rename the src/ruleset-wordpress.xml file to phpcs.xml.dist (in the project root).
That way PHPCS will pick up on it automatically and you don't need to pass the --standard.

@jrfnl While I agree, I am going to leave this as-is for now. I talked to the original author and they said it was for the site compatibility test, but it looks like it has morphed into what it is.

I don't want to get out-of-scope right now.